pkg/cortex/serve/cortex_internal/serve/serve.py

```diff
@@ -180,6 +181,10 @@ async def parse_payload(request: Request, call_next):
 
 
 def handle(request: Request):
+    result = async_to_sync(request.is_disconnected)()
```

Comment:
Nit: "result" could be renamed to something more descriptive, or this line and the if statement on the next line could be merged (so the result would no longer be assigned)